Unix/buildtool

```diff
@@ -605,6 +607,48 @@ fi
 
 ##==============================================================================
 ##
+## size command:
+##
+##==============================================================================
+
+if [ "$arg1" = "size" ]; then
+
+    if [ "$argc" != "1" -o "$opts" != "" ]; then
+        echo "Usage: $0 $arg1"
+        echo
+        exit 1
+    fi
+
+    case "$platform" in
+
+        LINUX_IX86_GNU|LINUX_X86_64_GNU)
+            echo size
+            ;;
+        MONTAVISTA_IX86_GNU)
+            echo size
+            ;;
+        NETBSD_IX86_GNU)
+            echo size
+            ;;
+        SUNOS_I86PC_SUNPRO|SUNOS_SPARC_SUNPRO)
+            echo echo 'size not avaliable'
```

Comment:
I think I get what's happening. But a comment to explain what you're doing and perhaps making this line `echo "echo 'size not available'"` might be more clear.